lua/metals.lua

```diff
@@ -34,31 +34,31 @@ M.analyze_stacktrace = function()
   end
 end
 
-M.ammonite_start = function()
+M.start_ammonite = function()
```

Comment:
So if we change these, we'd need to also change them under the Lua API section in the docs.

I like this, but also realize that this will break any mappings people may have that are using these. Let's put a note in the pr description that notifies people that this _does_ technically break the Lua API `nvim-metals` provides and also a mapping of `old -> new` to so it's crystal clear for someone.

Reply:
Documentation is changed.
I need to think about how correctly inform about changed commands.
I think fugitive has a nice way to inform about deprecated mappings.

Reply:
I've added deprecated and not supported variants for the commands.

From tomorrow till 28 June I am on vacation, so, unfortunately, I can't make any changes to this PR before 28.
So if this solution is not mature enough, then I'll pick it in 2 weeks. :)